### PR TITLE
Ind 5643 Google CUD recommendation type change

### PR DESCRIPTION
**Pull request by lawrethao**
### Description

changes recommendation type from usage reduction to a rate reduction recommendation

### Issues Resolved

Solves issue of incorrect categorization of policy

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt [hunk 1]
```diff
@@ -34,13 +35,6 @@ parameter "param_regions" do
   description "Location to check, this is region names"
 end
 
-parameter "param_automatic_action" do
```
Comment (nia-vf1) [inline cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt:37]: Should this PR include README changes as well? 

i.e., If this has been removed from the template, shouldn't this also be removed from the Input Parameters listed in the README?

File: cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt [hunk 2]
```diff
@@ -34,13 +35,6 @@ parameter "param_regions" do
...
-  type "list"
-  label "Automatic Actions"
-  description "When this value is set, this policy will automatically take the selected action(s)"
-  allowed_values ["Delete IP Addresses"]
-end
-
 parameter "param_log_to_cm_audit_entries" do
```
Comment (nia-vf1) [inline cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt:38]: This parameter isn't reflected in the README either

File: cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt (hunk 1 above)
Comment (lawrethao) [reply cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt:37]: this parameter isn't reflected in the readme.

**Review by nia-vf1 — APPROVED** [review]
LGTM

**Pull request merged**